assimilation_code/modules/assimilation/adaptive_inflate_mod.f90

```diff
@@ -29,8 +29,10 @@ module adaptive_inflate_mod
           get_is_prior,             get_is_posterior,       do_ss_inflate,      &
           set_inflation_mean_copy,  set_inflation_sd_copy,  get_inflation_mean_copy, &
           get_inflation_sd_copy,    do_rtps_inflate,        validate_inflate_options, &
-          print_inflation_restart_filename
-          
+          print_inflation_restart_filename, &
+          PRIOR, POSTERIOR, NO_INFLATION, OBS_INFLATION, VARYING_SS_INFLATION, &
```

Comment:
prior and posterior are pretty generic names to be in a public list.  if they're specifically for the arrays of 2 arguments in the inflation namelist items, i'd suggest they be PRIOR_INF and POSTERIOR_INF.  PRIOR and POSTERIOR have lots of meanings to filter in many other contexts other than inflation (state copies, for example) and making these public might end up with making confusing collisions.